src/Microsoft.AspNet.Mvc.TagHelpers/MissingAttributeLoggerStructure.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Framework.Logging;
+
+namespace Microsoft.AspNet.Mvc.TagHelpers
+{
+	internal class MissingAttributeLoggerStructure : ILoggerStructure
```

Comment:
There is not need for this type to be public at all, that's why the interface is so generic. I'll move it to an `Internal` namespace.